src/Files.Uwp/Helpers/FileExtensionHelpers.cs

```diff
@@ -48,12 +48,23 @@ public static bool IsPowerShellFile(string fileExtensionToCheck)
         /// otherwise <c>false</c>.</returns>
         public static bool IsZipFile(string fileExtensionToCheck)
         {
+            if (string.IsNullOrEmpty(fileExtensionToCheck))
+            {
+                return false;
+            }
+
             return new[] { ".zip", ".msix", ".appx", ".msixbundle", ".7z", ".rar", ".tar" }
                 .Contains(fileExtensionToCheck, StringComparer.OrdinalIgnoreCase);
         }
 
         public static bool IsBrowsableZipFile(string filePath, out string ext)
         {
+            if (string.IsNullOrEmpty(filePath))
+            {
+                ext = null;
```

Comment:
Can this be null? (you could always use string.Empty)

Reply:
If the function returns false, you know that the `out` parameter will be invalid. The actual value of `ext` in that case does not matter.